src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java

```diff
@@ -195,42 +191,6 @@ private static WebDriver createNewWebDriver(String forUserPersona, String browse
         return driver;
     }
 
-    @NotNull
-    private static DriverManagerType setupBrowserDriver(TestExecutionContext testExecutionContext,
-                                                        String browserType) {
-        DriverManagerType driverManagerType = DriverManagerType.valueOf(browserType.toUpperCase());
-        String webDriverManagerProxyUrl = (null == Runner.getWebDriverManagerProxyURL()) ? "" : Runner.getWebDriverManagerProxyURL();
-        LOGGER.info(String.format(
-                "Using webDriverManagerProxyUrl: '%s' for getting the WebDriver for browser: '%s'",
-                webDriverManagerProxyUrl, browserType));
-
-        // TODO - get browser version from local or container. What about cloud?
-        String browserVersion = NOT_SET;
-        if (Runner.isRunningInCI() && getCloudNameFromCapabilities().equalsIgnoreCase("docker")) {
-            LOGGER.info("Running in docker. Get driver for browser in docker");
-            String[] getBrowserVersionFromDockerCommand = new String[] {"cURL -s --request GET " +
-                                                        "'http://localhost:" + Runner.getRemoteDriverGridPort() + "/status'" +
-                                                       " | jq -r -M  '.value" +
-                               ".nodes[].slots[].stereotype | select(.browserName == \"" + browserType + "\") " +
-                                                                "| .browserVersion '"};
-            CommandLineResponse commandLineResponse = CommandLineExecutor.execCommand(getBrowserVersionFromDockerCommand);
-            browserVersion = commandLineResponse.getStdOut().split("\n")[0];
-            LOGGER.info(String.format("%s browser version in docker container: %s", browserType, browserVersion));
-        }
-        WebDriverManager webDriverManager = WebDriverManager.getInstance(driverManagerType)
-                                                            .proxy(webDriverManagerProxyUrl);
```

Comment:
if Proxy is not required, we need to remove it from the config.properties as well - WEBDRIVER_MANAGER_PROXY_KEY, right?

Reply:
Proxy is not required and thus all the unused methods and properties are removed now.